pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -195,8 +196,9 @@ def find_infectees_Covidsim(self, infector: Person,
         while number_to_infect > 0 and count < self._population.total_people():
             count += 1
             # Weighting for cell choice in Covidsim uses cum_trans and
-            # invCDF arrays, but really can't see how these are
-            # initialised. Have used the number of susceptible for now.
+            # invCDF arrays, which are equivalent to weighting by total
+            # susceptibles*max_transmission. May want to add transmission
+            # parameter later.
```

Comment:
What transmission parameter out of interest? I am currently adding an equivalent of `P.HouseholdTrans` from CovidSim, and intned to use equivalents for place etc

Reply:
max transmission does seem to be entirely dependent on the mininum distance between cells. You seem to think this makes sense. I don't.